src/modal/modal-config.ts

```diff
@@ -95,6 +95,15 @@ export interface NgbModalOptions {
 */
 @Injectable({providedIn: 'root'})
 export class NgbModalConfig implements NgbModalOptions {
+  ariaLabelledBy?: string;
```

Comment:
Please implement `Required<...>` for consistency and so we could avoid these problems in future:

```ts
export class NgbModalConfig implements Required<NgbModalOptions> {
  ariaLabelledBy: string;
  backdrop: boolean | 'static' = true;
  backdropClass: string;
  beforeDismiss: () => boolean | Promise<boolean>;
  centered: boolean;
  container: string;
  injector: Injector;
  keyboard = true;
  scrollable: boolean;
  size: 'sm' | 'lg' | 'xl';
  windowClass: string;
}
```

Reply:
Done